src/lib.rs

```diff
@@ -606,7 +612,7 @@ pub fn ticks_per_second() -> std::io::Result<i64> {
     if cfg!(unix) {
         match unsafe { sysconf(_SC_CLK_TCK) } {
             -1 => Err(std::io::Error::last_os_error()),
-            x => Ok(x.into()),
+            x => Ok(x),
```

Comment:
Can you leave this as `x.into()`?  Removing this will break the build on 32-bit, since on 32-bit linux, `sysconf` returns an `i32`.  So the `.x.into()` will widen this to an `i64`.  Same comment applies below to line 671.